src/ui/range-slider/types.ts

```diff
@@ -0,0 +1,7 @@
+export type RangeSliderValue =
+  | [number | null | undefined, number | null | undefined]
+  | [number | null | undefined]
+
+export type RangeSliderMode = 'single' | 'range'
+
+export type RangeSliderScale = 'linear' | 'logarithmic'
```

Comment:
better to create enums for those
we use `Glb.makeEnum`

Reply:
done